NuGet.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
OK that's fine. I'll add it then if needed :)

Thanks for this pull request BTW.